site/docs/Contributing/making-icons.mdx

```diff
@@ -0,0 +1,96 @@
+Orthogonals refer to keylines that intersect the center point of the icon and create additional vertices to use. These lines commonly slice the canvas at 90° and 45°. When further angles are needed increments of 15° and 5° are the next steps.
+
+## Masks
+> **Example: Basic steps to masking in most programs.**
+<Image src="../../static/assets/image/masking@2x.jpg" alt="Basic steps to masking" style="max-width:600px;" viewable />
+A mask customizes the container of an icon from the default square canvas. They also work great for testing different design ideas without having to commit to certain dimensions or layouts. Masks may be embedded in the asset itself or applied afterwards. 
+
+Note that some programs cut away the masked area, and some leave the masked area in place. Similarly, some programs need the mask layer to be on top and some on the bottom.
+
+Masks should always be flattened before exporting the icon asset.
+
+## About grids
+Grids are meant to be used as guides, not hard rules. Drop them when they stop working for you. Always check for optical balance. If something doesn’t feel quite right, trust your eyes more than the grid. 
+
+eg. The human eye (or the brain) is more tuned to seeing contrast in horizontal lines than vertical, so one should sometimes consider beefing up vertical lines.
+
+## Pixel grid
+> **Example: Pixel grid used for making HDS icons.**
+<Image src="../../static/assets/image/pixelgrid@2x.jpg" alt="A simple pixel grid for making icons." style="max-width:600px;" viewable />
+A pixel grid helps you draw in specific increments when snapping to a grid. A 1px increment has long held the standard in digital and an increment of 2px–8px has been adopted more recently. Icons sit snugly on their 24 x 24px system icon grid, using 1px–2px strokes. Pixel-snapping helps you render sharper icons on lower resolution screens but has become much less of a requirement as hardware rendering has improved.
+
+## Icon grid
+> **Example: A typical icon grid used for making icons.**
+<Image src="../../static/assets/image/icongrid@2x.jpg" alt="A simple icon grid for making icons." style="max-width:600px;" viewable />
+An icon grid is a tool for speed and consistency. When a drawing HDS icons, know the rules intimately and know when and how to bend them. The icon grid provides the same starting point for multiple contributors.
+
+## Safe area & trim area
+> **Example: The difference between safe area and trim area explained.**
+<Image src="../../static/assets/image/safearea_trimarea@2x.jpg" alt="The difference between safe area and trim area explained." style="max-width:600px;" viewable />
+The safe area or live area shows where the important content of the icon should live, while the inverse— the trim area—shows the area to avoid. In some cases the safe area is a soft guidance, but when the content gets cropped, the safe area becomes more crucial.
```

Comment:
> while the inverse— the trim area—shows the area to avoid

Puuttuuko tästä jotain? Välilyöntien puute tekee tästä vähän vaikean tulkita. Ehkä jättäisin joko "the trim area" osuuden pois (jos nyt siis ymmärsin lauseen oikein).